liquibase-core/src/test/java/liquibase/database/core/PostgresDatabaseTest.java

```diff
@@ -123,4 +126,33 @@ public void test_getConcatSql() {
         assertEquals("one || two || | three", database.getConcatSql("one", "two", "| three"));
     }
 
+    @Test
+    public void generatePrimaryKeyName_tableSizeNameLessThan63Bytes_nameIsBuiltCorrectly() {
+
+        // Arrange
+        final String tableName = "name";
+        final String expectedPrimaryKeyName = "name_pkey";
+
+        // Act
+        final String pk = this.database.generatePrimaryKeyName(tableName);
+
+        // Assert
+        assertEquals(pk, expectedPrimaryKeyName);
+    }
+
+    @Test
+    public void generatePrimaryKeyName_tableSizeNameMoreThan63Bytes_nameIsBuiltCorrectly() {
```

Comment:
I think it's worth adding some testing for non-ASCII characters.

Reply:
Some non-ASCII characters in tests would be good.